### ISSUE TITLE
Tracking how position of themes in the Design Picker grid impacts user selection

### ISSUE THREAD

**Issue by autumnfjeld** [Automattic/wp-calypso#68463]
# Description
This is an issue I'd like someone on the team to take (not me 🤣). It's a straightforward analysis.

First we'll ship the theme-with-styles such that they appear in a random order in the DP grid.    Soon after (say 7-10 days) we'll ship a change #68183 that will put theme-with-styles towards the top of the DP, so they get more prominent position.

This will give us a cool opportunity to compare each of these 8 themes (I think it is 8) before and after.   We'll probably want to wait a week after #68183 ships and then we can P2 the results.  Though analysis can start before then to get a feel for data and to ensure we don't have bugs in tracks events.

A really thorough analysis would include
* Data on theme preview/selection before we ship the styles feature
* Data in the period between feature ship and  #68183 
* Data after shipping  #68183 

This would encompass two comparisons (hhhhmmm, better to do two different P2 or combine these?):
* the impact of style selection feature
* the impact of position in the DP

@Automattic/ganon

**Comment by taipeicoder** [comment]
>Data on theme preview/selection before we ship the styles feature
Data in the period between feature ship and https://github.com/Automattic/wp-calypso/issues/68183

P2 link → pbxlJb-2x4-p2

**Comment by taipeicoder** [comment]
Closing in favor of #68463
